examples/multipole.py

```diff
@@ -0,0 +1,145 @@
+# Example illustrating the novel boundary sweeping functions (extrude/revolve) introduced in v0.8.0
+# at the hand of a simple simulation of a multipole
```

Comment:
A multipole is a very general term. What you are trying to make here is a stigmator [1]. Please rename the file to `stigmator.py`

[1] https://en.wikipedia.org/wiki/Stigmator

Reply:
changed.